packages/network-controller/CHANGELOG.md

```diff
@@ -6,6 +6,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [8.0.0]
+### Changed
+- Update EIP-1559 compatibility during network lookup ([#1236](https://github.com/MetaMask/core/pull/1236))
+- Refresh network when connection is reset ([#1235](https://github.com/MetaMask/core/pull/1235))
```

Comment:
We can drop this entry as well, since it's a change to a method that was also added in this release

```suggestion
```